sklearn/feature_selection/_sequential.py

```diff
@@ -30,11 +30,17 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin,
     estimator : estimator instance
         An unfitted estimator.
 
-    n_features_to_select : int or float, default=None
-        The number of features to select. If `None`, half of the features are
-        selected. If integer, the parameter is the absolute number of features
-        to select. If float between 0 and 1, it is the fraction of features to
-        select.
+    n_features_to_select : int or float, default='auto'
+        The number of features to select. If `'auto'`, features are selected
+        until score improvement does not exceed `tol`. If integer, the
+        parameter is the absolute number of features to select. If float
+        between 0 and 1, it is the fraction of features to select.
```

Comment:
Below we should add

```
.. versionadded:: 1.0
    The default changed from None to 'warn' in 1.0 and will become 'auto' in 1.2.
    To keep the same behaviour as with `None`, you should manually set
    `n_features_to_select='auto'` and set `tol=None`
```

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20145/commits/9b11fbbcfb1b443063a25940f4f8d3f5a4c6951e